packages/fuels-core/src/json_abi.rs

```diff
@@ -342,6 +351,13 @@ impl ABIParser {
             return Ok(Token::Array(vec![]));
         }
 
+        //To parse the value string we use a two pointer/index approach.
+        //The items are comma separated and if an item is tokenized the last_item
+        //index is moved to the current position.
+        //The variable nested is incremented and decremented if a bracket is encountered,
+        //and appropraite errors are thrown if the nested count is not 0
+        //Characters inside quotes are ignored and they are tokenized as one item.
+        //An error is return if there is an odd number of quotes.
```

Comment:
I think you can say something along the lines of `for more details about this algo, refer to the tokenize_struct method` so we can avoid having to update the docstring everytime :smiley:

Reply:
Done

Reply:
I don't see it done, I guess you forgot to push?

Reply:
Now it should be ok.